pkg/configmap/configmap_test.go

```diff
@@ -77,6 +77,19 @@ func TestLoad(t *testing.T) {
 				assert.Equal(t, 3, len(objects))
 			},
 		},
+		{
+			name:   "BundleWithNoDefaultChannel",
+			source: "testdata/bundle-with-no-default-channel.yaml",
+			assertFunc: func(t *testing.T, bundleGot *api.Bundle) {
+				csvGot := bundleGot.GetCsvJson()
+				assert.NotNil(t, csvGot)
```

Comment:
Nit: Pointless assertion, `csvGot` is a string. Actually, two tests above this one asserts nothing but this.